DER/DerivativesContracts/Forwards.rdf

```diff
@@ -260,113 +434,169 @@
-		<rdfs:label xml:lang="en">cash delivery</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;CashCloseout"/>
-		<rdfs:range rdf:resource="&xsd;boolean"/>
-	</owl:DatatypeProperty>
-	
-	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;cashDelivery.1">
-		<rdfs:label xml:lang="en">cash delivery</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardDeliveryCommitment"/>
-		<rdfs:range rdf:resource="&xsd;boolean"/>
-	</owl:DatatypeProperty>
-	
 	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;closedOutBy">
 		<rdfs:label xml:lang="en">closed out by</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-der-der-fwd;CashCloseout"/>
-		<rdfs:range rdf:resource="&fibo-der-der-fwd;ForwardContract"/>
+		<rdfs:range rdf:resource="&fibo-der-der-fwd;Forward"/>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;deliveryAmount">
-		<rdfs:label xml:lang="en">delivery amount</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardCashDeliveryCommitment"/>
-		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;firstDeliveryDate">
+		<rdfs:label xml:lang="en">first delivery date</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Future"/>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
+		<skos:definition xml:lang="en">The start date of the range of dates by which the underlying for a futures contract must be delivered in order for the terms of the contract to be fulfilled.</skos:definition>
 	</owl:ObjectProperty>
 	
-	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;deliveryDate">
-		<rdfs:label xml:lang="en">delivery date</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardCashDeliveryCommitment"/>
-		<skos:definition xml:lang="en">FpML = valueDate Definition for &apos;valueDate&apos; in FpML &apos;The date on which both currencies traded will settle.&apos; Definition for &apos;currency1ValueDate&apos; in FpML &apos;The date on which the currency1 amount will be settled. To be used in a split value date scenario.&apos; Notes 20 Jan: single value date scenario adds nothing to the model. The &quot;split value date&quot; is when Payment Date and SettlementDate are different; otherwise they are the same. We don&apos;t need to model this.</skos:definition>
-	</owl:DatatypeProperty>
+	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;firstNoticeDate">
+		<rdfs:label xml:lang="en">first notice date</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Future"/>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
+		<skos:definition xml:lang="en">The first date at which a delivery notice can be issued.</skos:definition>
+	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;forwardPrice">
-		<rdfs:label xml:lang="en">forward price</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardDeliveryCommitment"/>
+	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;hasAdditionalCosts">
```

Comment:
Should be singular, unless this represents a set of costs in which case Set or Collection should be in the name